recipes/festival/build.sh

```diff
@@ -0,0 +1,44 @@
+set -exou
+
+# The festival file system hierarchy is insane; here we correct a few preconfigured folders
+
+sed -i "s#^EST=.*\$#EST=${BUILD_PREFIX}/lib/speech_tools#" config/config.in  # Path to libestools
+sed -i "s#^FTLIBDIR =.*\$#FTLIBDIR = ${PREFIX}/lib#" config/project.mak  # Where the installed libraries AND dictionaries/voices can be found
+sed -i "s#^INSTALL_PREFIX=.*\$#INSTALL_PREFIX=${PREFIX}#" config/systems/default.mak
+sed -i "s#\$(FESTIVAL_HOME)/bin/festival#${PREFIX}/bin/festival#" examples/Makefile
+sed -i "s#/projects/festival/lib/dicts#${PREFIX}/share/festival/dicts#" lib/lexicons.scm  # Default path for dictionaires (can be overriden with --libdir).
+
+./configure
+
+# This make call just generates a file on which we need to apply the following sed commands, too
+make modules
+(cd src/modules; make init_modules.cc)
+
+# The include statements do not follow any filesystem hierarchy. This command straightens that up. Let's pray it is correct!
```

Comment:
You can check your built package to see if it is correct. Download the build artifacts on the Azure Pipelines page.

Reply:
The result looks okay. These seds were taken from Arch linux: https://gitlab.archlinux.org/archlinux/packaging/packages/festival/-/blob/main/PKGBUILD?ref_type=heads#L127 .